### PR TITLE
Add Trino 373 release notes

### LINKED ISSUE #11288: Release notes for 373

### PR DESCRIPTION
**Pull request by mosabua**
## Description

Assemble the release notes for the upcoming Trino 373 release

> Is this change a fix, improvement, new feature, refactoring, or other?

Improvement.

> Is this a change to the core query engine, a connector, client library, or the SPI interfaces? (be specific)

Documentation only.

> How would you describe this change to a non-technical end user or system administrator?

Add the release notes for Trino 373 with sections for general, security, connectors and other topics as included in the release. 

## Related issues, pull requests, and links

* Fixes #11288
* Follow up to  #11094

## Documentation

( ) No documentation is needed.
(✅ ) Sufficient documentation is included in this PR.
( ) Documentation PR is available with #prnumber.
( ) Documentation issue #issuenumber is filed, and can be handled later.

## Release notes

(✅ ) No release notes entries required.
( ) Release notes entries required with the following suggested text:

##  Verification for each commit

Format: commit, committer/maintainer, PR/issue number, ✅ / ❌  RN, ✅ / ❌   docs
✅  RN - release note added and verified, or assessed to be not necessary, set to ❌ RN before completion
✅  docs - need for docs assessed and merge, or assessed to be not necessary, set to ❌ docs before completion

## 2 Mar 2022 ✅ 

* https://github.com/trinodb/trino/commit/628b146cdbe4abac769749278741ce86353c0671, @Praveen2112 #10215, ✅ RN ✅ docs  
* https://github.com/trinodb/trino/commit/e07c7c3e74e71de55445a97841508a30b94a4819, @Praveen2112 #10215 ✅ RN  ✅ docs  
* https://github.com/trinodb/trino/commit/fe230f6a0315732d7a8be4d3af9a7d9e82896b1a, @Praveen2112 #10215 ✅ RN ✅ docs  
* https://github.com/trinodb/trino/commit/2842a392d7e46f14c8b6898becafd342c4178d3a, @hashhar #11274 ✅ RN ✅ docs 
* https://github.com/trinodb/trino/commit/b67d9bbfad2e9490e7bd826c42bac0a984d2880d, @hashhar #11274 ✅ RN ✅ docs  
* https://github.com/trinodb/trino/commit/4b20ba17d3c86a334a179196c1b5536d823f103c, @hashhar #11274 ✅ RN ✅ docs  

## 3 Mar 2022 ✅ 

* https://github.com/trinodb/trino/commit/d6729447c73823cb095b354e4f4e79c02e5488ec @hashhar #11250 ✅ RN ✅ docs 
* https://github.com/trinodb/trino/commit/1f192655cebe9e639f81d23c180c0f69617de9d3 @hashhar #11250 ✅ RN ✅ docs 
* https://github.com/trinodb/trino/commit/dacde64ef28845bfde821e21761bb2adf4abb210 @hashhar #11250 ✅ RN ✅ docs 
* https://github.com/trinodb/trino/commit/41c3ae79b278b860c94ecd817b2b412160c79ec4 @hashhar #11250 ✅ RN ✅ docs 
* https://github.com/trinodb/trino/commit/d92fcf36adfa57311c5145589c6e3d2c859bfb29 @findepi #11219 ✅ RN ✅ docs 
* https://github.com/trinodb/trino/commit/63845b163ef30972018783262b73039e81007a1f @findepi #11219 ✅ RN ✅ docs 
* https://github.com/trinodb/trino/commit/29905897400fea8b0411ca1034196d6e7e820f11 @findepi #11219 ✅ RN ✅ docs 
* https://github.com/trinodb/trino/commit/63d965bd8a7b9816a23909c80935e80f7dafa949 @findepi #11219 ✅ RN ✅ docs 
* https://github.com/trinodb/trino/commit/bfef0bf458ad6b8284fecadd7d554349f33844df @findepi #11062 ✅ RN ✅ docs
* https://github.com/trinodb/trino/commit/aa2c81c7434d6b315739f18593b42e42f09bb6ff @findepi #11062 ✅ RN ✅ docs 
* https://github.com/trinodb/trino/commit/73625a1872fb24af80a3e06581d6377b63f949c8 @findepi #11062 ✅ RN ✅ docs
* https://github.com/trinodb/trino/commit/6c8b0f310685b35cdaabf57f186c7afdd1cd86de @findepi #11254  ✅ RN ✅ docs
* https://github.com/trinodb/trino/commit/a8da430d2d165cd595a3275dd41ee57ede64a179 @sopel39 #11286 ✅ RN ✅ docs
* https://github.com/trinodb/trino/commit/5dc97c35743bcf45cb06679d9df7706d360ebeab, @martint #11259 ✅  RN ✅  docs 
* https://github.com/trinodb/trino/commit/132452ec545495f5dcdef4e3da6cbeb23d3a2af2, @losipiuk #11255 ✅ RN ✅ docs
* https://github.com/trinodb/trino/commit/d146e11dbed96426cdfdcb9f9640c58cc3a624e9, @ebyhr #11272 ✅ RN ✅ docs
* https://github.com/trinodb/trino/commit/9cf6806684afe678db703ec86ded859ff378f971, @hashhar #11313 ✅ RN ✅ docs
* https://github.com/trinodb/trino/commit/bef3b2854c0b8a459e2659f2fbed5b1daa66f907, @ebyhr #11319 ✅ RN ✅ docs

## 4 Mar 2022  ✅ 

* https://github.com/trinodb/trino/commit/d102e33383798e218215d9ad52a0bbf9044bb87f @findepi #11045 ✅ RN ✅  docs 
* https://github.com/trinodb/trino/commit/b6eba7721882b516007316cf24306bc7d9a1f94d @findepi #11045 ✅ RN ✅  docs
* https://github.com/trinodb/trino/commit/ebbcc0c13bd4b0247eb1ed70d9c176e3f6bbd35a @findepi #11045 ✅ RN ✅  docs
* https://github.com/trinodb/trino/commit/a58bb8e44516d534fb6903939d5845be645e9664 @findepi #11045 ✅  RN ✅  docs
* https://github.com/trinodb/trino/commit/d1ff5157e2f822b616daaf2fe8c9a9e555131aef @ebyhr #11291 ✅ RN✅ docs
* https://github.com/trinodb/trino/commit/8b06b028674f1a2752ded672ba667321247bb77d @sopel39 #11146 ✅ RN✅ docs
* https://github.com/trinodb/trino/commit/6adcfd8ef35585dbbef1736ad65b171a9e6da797 @sopel39 #11146 ✅ RN✅ docs
* https://github.com/trinodb/trino/commit/1fbe4f590af25a1589e7e94322ed0ff36ef7c146 @sopel39 #11146 ✅ RN✅ docs
* https://github.com/trinodb/trino/commit/1de0bb9d07cc0645575eec7b723e44eaffe85459 @sopel39 #11146 ✅ RN✅ docs
* https://github.com/trinodb/trino/commit/a86e13b924c6155499b5794be0dccb2f87a113e6 @findepi #10844 ✅ RN✅ docs
* https://github.com/trinodb/trino/commit/e12a9b139a53cd1876ef04ae06bc14c290e18e36 @findepi #10844 ✅ RN✅ docs
* https://github.com/trinodb/trino/commit/f712fe73e27d797a0a50c36121afa26cf3206906 @findepi #10844 ✅ RN✅ docs
* https://github.com/trinodb/trino/commit/902f4b2cfa3032af2fa47aed99946211a97bf0b1 @findepi #10844 ✅ RN✅ docs
* https://github.com/trinodb/trino/commit/c55bc4f8a96ea343bd8d3c81a83f6a853a2c0aba @findepi #10844 ✅ RN ✅ docs
* https://github.com/trinodb/trino/commit/09287de52a42b8c8b73aa549741514172bd453ee @electrum #11232 ✅ RN ✅ docs
* https://github.com/trinodb/trino/commit/1822cfa45a3f4b2ba6770b6efb0877af25de4076 @electrum #11232 ✅ RN ✅ docs
* https://github.com/trinodb/trino/commit/cbd5ca34cb4a4305ee8ab9621aced40f89e7a096 @electrum #11026 ✅ RN ✅ docs
* https://github.com/trinodb/trino/commit/92b458c30a01aa997c68c9d898998e1c1b8076a4 @electrum #11329  ✅ RN ✅ docs
* https://github.com/trinodb/trino/commit/88a1be873e1c429ff4a066d9031a24c2716d0fa9 @electrum #11328 ✅ RN ✅ docs
* https://github.com/trinodb/trino/commit/108628bc33605c49d5c214150bd9c30933358fb1 @martint #11312 ✅ RN ✅ docs

## 5 Mar 2022 ✅ 

* https://github.com/trinodb/trino/commit/26a176fe2d02984ef4688b07dfc90183f59fac77 @findepi  #11309 ✅ RN ✅ docs
* https://github.com/trinodb/trino/commit/4175b09589c79ad888dfcb2805bb1d85c0cf347f @findepi #11309 ✅ RN ✅ docs
* https://github.com/trinodb/trino/commit/45e0ecb796fc289df6125c2397e6ff18525559eb @findepi #11309 ✅ RN ✅ docs
* https://github.com/trinodb/trino/commit/ee5c3f2937999fd958642449b095834aea7b5a1d @findepi #11309 ✅ RN ✅ docs
* https://github.com/trinodb/trino/commit/1568a7655dc859dbe23aa64f6c1aa723f6eea028 @findepi #11163✅ RN ✅ docs
* https://github.com/trinodb/trino/commit/c9920a278a73b781665a061ad82e694c2f0ad349 @findepi #11163✅ RN ✅ docs
* https://github.com/trinodb/trino/commit/c3ff79950513599ab84a70919f93dbb2ae9d9b71 @findepi #11163✅ RN ✅ docs
* https://github.com/trinodb/trino/commit/15e7374ffe4b663f052b3c0d983363d67dc3abe1 @findepi #11163✅ RN ✅ docs

## 6 Mar 2022 ✅ 
 
* https://github.com/trinodb/trino/commit/5c87f42c3621a4c6108f81891f61fe080e40e8e9 @ebyhr #11320 ✅ RN ✅ docs
 
## 7 Mar 2022 ✅ 

* https://github.com/trinodb/trino/commit/98b794a9f7228619f4f8feb2c9bbb4d75d70b8f5 @ebyhr #11212 ✅ RN ✅ docs
* https://github.com/trinodb/trino/commit/98babf15b8de3d784bffaadae3e98458e9e9c9e4 @ebyhr #11340 ✅ RN ✅ docs
* https://github.com/trinodb/trino/commit/4e133424b2d4bc52cd3b65f391d19e480168054b @losipiuk #11331 ✅ RN ✅ docs
* https://github.com/trinodb/trino/commit/d4417ea744c631e244d82ae3364b4fb45cfa22ea @sopel39 #10965 ✅ RN ✅ docs
* https://github.com/trinodb/trino/commit/82b4c471ad8e4871dd867e4d72dc8eb2ffd6cd04 @sopel39 #10965  ✅ RN ✅ docs
* https://github.com/trinodb/trino/commit/7a96ffc421d8c4d82804a72582223e7dc0033724 @electrum #11022 ✅ RN ✅ docs
* https://github.com/trinodb/trino/commit/8a1c4d72af310ef8d9315aa8bfb7620522005294 @hashhar #11357  ✅ RN ✅ docs

## 8 Mar 2022 

* https://github.com/trinodb/trino/commit/9aeb65dffcb355016ca1ce0a1f7f060e88feab84 @findepi #11354 ✅ RN ✅ docs
* https://github.com/trinodb/trino/commit/73c18ac37227337d7f8281ba6db08a38e017b6d4 @ebyhr #11351 ✅ RN ✅ docs
* https://github.com/trinodb/trino/commit/eb377853b32e286f7c30116d4886a365de087e67 @sopel39 #11353 ✅ RN ✅ docs
* https://github.com/trinodb/trino/commit/9956543e429419a268cfe20a47af7f7c47fa2398 @findepi  #10897 ✅ RN ❌  docs, https://github.com/trinodb/trino/pull/11229
* https://github.com/trinodb/trino/commit/84f0b694c27b7ae20b94c0cf4d9d0f745c774f52 @sopel39 #11333 ✅ RN ✅ docs
* https://github.com/trinodb/trino/commit/c6fd4e4e6a834418dab4594fa338dba822f93442 @losipiuk #11335 ✅ RN ✅ docs
* https://github.com/trinodb/trino/commit/495868863d1dd687251dfda27b1c500d8fea600f @losipiuk #11335 ✅ RN ✅ docs
* https://github.com/trinodb/trino/commit/23608a795b0a0cf10b68bfa24b97819b3f2a9594 @losipiuk #11327 ✅ RN ✅ docs
* https://github.com/trinodb/trino/commit/88e435ceb0ee65cc3704f01f73ae3a26595b0c10 @losipiuk #11327 ✅ RN ✅ docs
* https://github.com/trinodb/trino/commit/e0d29a81fedb20d32f2d96c9f38c11ec4d30967b @losipiuk #11327 ✅ RN ✅ docs
* https://github.com/trinodb/trino/commit/90ea68bad24f31017b9c6424432365e13c591945 @losipiuk #11129 ✅ RN ✅ docs
* https://github.com/trinodb/trino/commit/0efc2b939dda8a4976a14fc8d97f92172eabcde9 @losipiuk #11129 ✅ RN ✅ docs
* https://github.com/trinodb/trino/commit/f26ee2b50966061f55d0e91c48eb4df180008646 @losipiuk #11129 ✅ RN ✅ docs
* https://github.com/trinodb/trino/commit/507724ec3ec16770d6410c03ade3b0372d515e08 @losipiuk #11129 ✅ RN ✅ docs
* https://github.com/trinodb/trino/commit/14f01f20ea50951804b41d229521c03747a70c79 @losipiuk #11129 ✅ RN ✅ docs
* https://github.com/trinodb/trino/commit/bafb4aca5b299f89295f5458c91a3449aa97f398 @losipiuk #11304 ✅ RN ✅ docs
* https://github.com/trinodb/trino/commit/d8f7ac8be3e9273ef556fc26ab7dc12759ddb956 @findepi #11159 ✅  RN ✅   docs
* https://github.com/trinodb/trino/commit/b75805a0e8ad467f94347b9390329ca607492239 @hashhar #11362 ✅ RN ✅ docs
* https://github.com/trinodb/trino/commit/6a63d25953e98571ff511bbe0a6de632e1919be4 @hashhar #11365  ✅ RN ✅ docs
* https://github.com/trinodb/trino/commit/e7e2a82f260ff75a3acdcec7de1b0ce93bc5d384 @hashhar #11375 ✅ RN ✅ docs
* https://github.com/trinodb/trino/commit/3bd82ba003bf019767e3c6fe99e42088d0915157 @hashhar #11374 ✅ RN ✅ docs

## 9 Mar 2022 

* https://github.com/trinodb/trino/commit/d83907be77c6c69ac24c1e2a2dbbcc5edf97f08e @ebyhr #11383 ✅ RN ✅ docs
* https://github.com/trinodb/trino/commit/c8419803bbd1d2e8d4aec9ad8f9376edec06ccf7 @ebyhr #11378 ✅ RN ✅ docs
* https://github.com/trinodb/trino/commit/8d0ff2216f8abdb7556cc48885f5782febb3bfc9 @findepi #11270 ✅ RN ✅ docs
* https://github.com/trinodb/trino/commit/c9f6e77e446251a2ce7e07dd595a5196e6527663 @losipiuk #11023 ✅ RN ✅ docs
* https://github.com/trinodb/trino/commit/9e957e4f8238f6e8a204c51e976c1ce911c1fb2e @hashhar #11385 ✅ RN ✅ docs
* https://github.com/trinodb/trino/commit/a3ed6badd16169db00e7c4fa1a6118bab1d1dadf @hashhar #11385 ✅ RN ✅ docs
* https://github.com/trinodb/trino/commit/788c5d6491a136c4d38db587856f449413935df6 @findepi  #11393 ✅ RN ✅ docs
* https://github.com/trinodb/trino/commit/1b7e4a758a53d34f2074348e62e71ec0af95d1df @findepi #11395 ✅ RN ✅ docs
* https://github.com/trinodb/trino/commit/fcde77f160d92212c13638f4a0652b57dc67d082 @phd3 #11226 ✅ RN ✅ docs
* https://github.com/trinodb/trino/commit/966f22080b46eaa5909d1dd038498c2ebf8b8172 @findepi #11240 ✅ RN ✅ docs
* https://github.com/trinodb/trino/commit/4bc5857e5ee7d551d6e4867cf110cc50534d84c1 @findepi #11308 ✅ RN ✅ docs
* https://github.com/trinodb/trino/commit/ccb3d307281b9d360f048cc7fa212bb718f20215 @findepi #11308 ✅ RN ✅ docs

### CODE REVIEW

File: docs/src/main/sphinx/release/release-373.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  communication. ({issue}`11146`)
+
+## BigQuery connector
+
+* Allow configuring view expiration time via the `bigquery.view-expire-duration`
+  config property. ({issue}`11272`)
+
+## Hive connector
+
+* Prevent failure for queries with the final number of partitions
```
Comment (mosabua) [inline docs/src/main/sphinx/release/release-373.md:15]: how about that @Praveen2112 ?

File: docs/src/main/sphinx/release/release-373.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+  communication. ({issue}`11146`)
+
+## BigQuery connector
+
+* Allow configuring view expiration time via the `bigquery.view-expire-duration`
+  config property. ({issue}`11272`)
+
+## Hive connector
+
+* Support access to S3 via a HTTP proxy. ({issue}`11255`)
```
Comment (mosabua) [inline docs/src/main/sphinx/release/release-373.md:15]: Is this rewording okay @losipiuk  ..

**Review by martint — CHANGES_REQUESTED** [review]

File: docs/src/main/sphinx/release/release-373.md [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+  `information_schema.columns` and a materialized view is removed during the
+  query. ({issue}`10976`)
+
+## Oracle connector
+
+* Fix query failure when performing concurrent write operations. ({issue}`11318`)
+
+## Phoenix connector
+
+* Prevent writing incorrect results by failing a query when the Phoenix JDBC
```
[the remark this answers is not in the document]
Comment (hashhar) [reply docs/src/main/sphinx/release/release-373.md:65]: Not under our control. It's a bug in driver. It replaces them unconditionally.

File: docs/src/main/sphinx/release/release-373.md [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+## General
+
+* Add Delta Lake connector (TBD .. link to docs once merged). ({issue}`11296`, {issue}`10897`)
+* Improve query performance by reducing overhead of cluster internal
+  communication. ({issue}`11146`)
+* Handle `varchar` to `timestamp` conversion errors in {func}`try`. ({issue}`11259`)
+* Add redirection awareness for `DROP COLUMN` task. ({issue}`11304`)
+* Add redirection awareness for `RENAME COLUMN` task. ({issue}`11226`)
+* Disallow table redirections in `SHOW GRANTS` statement. ({issue}`11270`)
+* Add CPU time, scheduled time, and cumulative memory statistics regarding
```
[the remark this answers is not in the document]
Comment (mosabua) [reply docs/src/main/sphinx/release/release-373.md:12]: done

File: docs/src/main/sphinx/release/release-373.md [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+  failed tasks in a query to query-completion events. ({issue}`10734`)
+* Add support for killing individual tasks of queries run with ``retry-mode``
```
[the remark this answers is not in the document]
Comment (mosabua) [reply docs/src/main/sphinx/release/release-373.md:14]: done

File: docs/src/main/sphinx/release/release-373.md [hunk 6]
```diff
@@ -0,0 +1,71 @@
...
+
+## BigQuery connector
+
+* Allow configuring view expiration time via the `bigquery.view-expire-duration`
+  config property. ({issue}`11272`)
+
+## Elasticsearch connector
+
+* Improve performance of queries involving `LIKE` by pushing predicate
+  computation the underlying database. ({issue}`7994`, {issue}`11354`,
```
[the remark this answers is not in the document]
Comment (mosabua) [reply docs/src/main/sphinx/release/release-373.md:34]: both addressed

**Pull request merged**